src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 800;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-bar {
+  position: relative;
+  margin-top: 20px;
+  box-sizing: border-box;
+}
+
+.search-bar__input {
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  flex: 1;
+  outline: none;
+  line-height: 68px;
+  padding: 0 0 0 62px;
+  font-family: inherit;
+  font-weight: inherit;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input:focus {
+  background: linear-gradient(
+    180deg, 
+    #fff 0%, 
+    #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir-Heavy, sans-serif;
+  font-size: 16px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.search-bar__input--small {
+  line-height: 40px;
+  padding: 0 0 0 33px;
+}
+
+.search-bar__input::placeholder {
+  font-family: Avenir, sans-serif;
```

Comment:
```suggestion
  font-family: inherit;
```

Reply:
Hi! I don't know how to solve this. Your suggestion didn't help...